main/com.tunein/index.js

```diff
@@ -71,16 +74,27 @@ module.exports = class TuneinRadioDevice extends Tp.BaseDevice {
         };
     }
 
-    async _http_get(url) {
+    async _http_get(url, content_type) {
         try {
-            return JSON.parse(await Tp.Helpers.Http.get(url, {dataContentType: CONTENT_TYPE}));
+            return JSON.parse(await Tp.Helpers.Http.get(url, {dataContentType: content_type}));
         } catch (e) {
             if (!e.detail)
                 throw e; 
             throw new Error(JSON.parse(e.detail).error.message);
         }
     }
 
+    async _http_post(url, data='', content_type=CONTENT_TYPE.json) {
+        return await Tp.Helpers.Http.post(url, data, {
+            dataContentType: content_type,
+            accept: content_type
```

Comment:
`accept: application/x-www-form-urlencoded` doesn't make sense. Also, do you ever call this method?